CHANGELOG.rst

```diff
@@ -177,7 +180,7 @@ v0.7.0 RC2
    * Added publications to bibliography (PR #1256)
    * Adding contributing guide and code of conduct (#1258)
    * Changed Doc and Tutorial mentions of iPython notebooks to Jupyter notebooks (PR #1262)
-    
```

Comment:
Are these automatic from your editor?

Reply:
I didn't think I'd changed anything here, just the spot on lines 81 and 82.

Reply:
Cool, it's not a problem. Neat that it autodeletes extra spaces!